src/Tgstation.Server.Host/Security/OAuth/InvisionCommunityOAuthValidator.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Net.Http;
+
+using Microsoft.Extensions.Logging;
+
+using Tgstation.Server.Api.Models;
+using Tgstation.Server.Host.Configuration;
+using Tgstation.Server.Host.System;
+
+namespace Tgstation.Server.Host.Security.OAuth
+{
+	/// <summary>
+	/// OAuth validator for Discord.
+	/// </summary>
+	sealed class InvisionCommunityOAuthValidator : GenericOAuthValidator
+		{
+		/// <inheritdoc />
+		public override OAuthProvider Provider => OAuthProvider.InvisionCommunity;
+
+		/// <inheritdoc />
+		protected override Uri TokenUrl => new Uri($"{BaseProtocolPath}/oauth/token/");
```

Comment:
```suggestion
		protected override Uri TokenUrl => new Uri($"{BaseProtocolPath}/oauth/token");
```

Reply:
Nada
![image](https://user-images.githubusercontent.com/25063394/181589203-fda80a5e-8a95-4441-ac39-cd972622c4e6.png)

Reply:
`System.Uri` handles the trailing slash so it's just a style thing

Reply:
Not a styling thing. It straight up breaks if you dont have it.